deploy/docker/LOCALHOST.SECURE.md

```diff
@@ -94,10 +94,18 @@ Edit all the fields according to your specific case.
   |:------------|:---------------|:---------------|
   | DTAAS_DIR | '/home/Desktop/DTaaS' | Full path to the DTaaS directory. This is an absolute path with no trailing slash. |
   | username1 | 'user1' | Your gitlab username |
-  | CLIENT_CONFIG | '/home/Desktop/DTaaS/deploy/config/client/env.local.js' | Full path to env.js file for client |
 
-:clipboard: The path examples given here are for Linux OS.
-These paths can be Windows OS compatible paths as well.
+<!-- markdownlint-disable MD046 -->
```

Comment:
The `tip` is not being rendered correctly in markdown viewer. Please revert to
`:clipboard: Important points to note:`